client/src/components/Navigation/RegisterLogin.js

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import {Button, Dialog, DialogContent, DialogTitle, AppBar, Tabs, Tab, Typography} from "@material-ui/core";
+import PropTypes from "prop-types";
+
+function TabContainer(props) {
+  return (
+    <Typography component="div" style={{ padding: 8 * 3 }}>
+      {props.children}
+    </Typography>
+  );
+}
+
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired
+};
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper
+  }
+});
+
+export default class RegisterLogin extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      open: false,
+      value: 0,
+      email: "",
+      passwordMessage: "",
+      password1: "",
+      password2: ""
+    };
+  }
+
+  handleClickOpen = () => {
+    this.setState({ open: true });
+  };
+
+  handleClose = () => {
+    this.setState({ open: false });
+  };
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+  inputchange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  submit = event => {
+    event.preventDefault();
+    const { password1, password2 } = this.state;
+    if (password1 !== password2) {
+      this.setState({
+        passwordMessage: "Passwords do not match!"
+      });
+    } else {
+      this.setState({
+        passwordMessage: "Passwords match!"
```

Comment:
I don't know if it's necessary to confirm that the passwords match. Usually, the confirmation is that you successfully register.